index.scss

```diff
@@ -25,9 +25,37 @@ body{
     top: 0;
     width: 100%;
     z-index: 10;
+    display: flex;
+    justify-content: space-between;
     }
     h1 {
       padding: 15px 0 15px 10px;
+    }
+  .homePageTitle {
+    text-decoration: none;
+  }
+  .signUp {
+    padding: 20px 7px;
+    margin: 0 20px;
+    display: block;
+    width: 60px;
+    height: 60px;
+    background-color: #ffffff;
+    border-radius: 50%;
+    font-size: 0.9rem;
+    text-decoration: none;
+    font-family: "Trebuchet MS", sans-serif;
+    color: #000000;
+    -webkit-transition: all 0.3s;
```

Comment:
why webkit transition? doesn't it work without?

Reply:
webkit transition has been removed from index.scss